### ISSUE TITLE
COMBINE API deployment not reading shared S3 bucket configuration

### ISSUE THREAD

**Issue by jonrkarr** [reproducible-biomedical-modeling/CRBM-Viz#3867]
This causes multiple things to fail
- Creation of COMBINE archives
- Export of visualizations to COMBINE archives

Here's the trace from the `dev` deployment. The problem appears to be that `secret.env` isn't mounted.
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.9/site-packages/connexion/decorators/decorator.py", line 68, in wrapper
    response = function(request)
  File "/usr/local/lib/python3.9/site-packages/connexion/decorators/uri_parsing.py", line 149, in wrapper
    response = function(request)
  File "/usr/local/lib/python3.9/site-packages/connexion/decorators/validation.py", line 193, in wrapper
    response = function(request)
  File "/usr/local/lib/python3.9/site-packages/connexion/decorators/parameter.py", line 115, in wrapper
    return function(**kwargs)
  File "/app/src/handlers/combine/create.py", line 161, in handler
    archive_url = src.utils.save_file_to_s3_bucket(archive_filename, public=True)
  File "/app/src/utils.py", line 80, in save_file_to_s3_bucket
    s3_bucket = get_s3_bucket()
  File "/app/src/utils.py", line 67, in get_s3_bucket
    s3_bucket = S3Bucket()
  File "/app/src/s3.py", line 12, in __init__
    self.validate_configuration(config)
  File "/app/src/s3.py", line 69, in validate_configuration
    raise ValueError('The configuration for the S3 bucket is not valid:\n  {}'.format('\n  '.join(errors)))
ValueError: The configuration for the S3 bucket is not valid:
  Storage endpoint (`STORAGE_ENDPOINT`) must be set, not `None`.
  Public storage access (`STORAGE_PUBLIC_ENDPOINT`) must be set, not `None`.
  Storage bucket (`STORAGE_BUCKET`) key must be set, not `None`.
```

**Comment by jonrkarr** [comment]
I think I figured it out. 
- I removed these configuration variables from the configuration for the COMBINE API because they're redundant with the shared configuration. 
- The COMBINE API wasn't reading the shared configuration.
- The configuration files are correctly getting mounted. We know this because it is correctly reading the bucket token and secret.
- I think my confusion stemmed from thinking I had tested our recent changes, but didn't because they hadn't actually been deployed.
- I changed the COMBINE API to also read the shared configuration file.

Should be fixed with #3871.
